test/wh_test_crypto.c

```diff
@@ -562,6 +561,32 @@ int whTest_CryptoClientConfig(whClientConfig* config)
         WH_ERROR_PRINT("Failed to wh_Client_KeyExport %d\n", ret);
         goto exit;
     }
+    printf("CMAC SUCCESS\n");
+    /* test counters */
+    keyId = 1;
+    if ((ret = wh_Client_CounterReset(client, keyId, &counter)) != 0 || counter != 0) {
+        WH_ERROR_PRINT("Failed to wh_Client_CounterReset %d\n", ret);
+        goto exit;
+    }
+    if ((ret = wh_Client_CounterIncrement(client, keyId, &counter)) != 0 || counter != 1) {
```

Comment:
Make sure to add a loop that invokes Increment more times than entries in the directory (currently 32), so this call will return with WH_ERROR_NOSPACE.  You should add a wh_Client_NvmDestroyObjects(client, 0,NULL,&out_rc) to reclaim the space and retry the CounterIncrement.  Just noticed the DestroyObjects API is incorrect.